source/includes/steps-install-shell-macos-homebrew.yaml

```
@@ -0,0 +1,33 @@
+---
+title: "Install Homebrew."
+ref: install-homebrew
+level: 4
+content: |
+
+   Refer to the `Homebrew <https://brew.sh/>`__ website
+   for instructions to install Homebrew on macOS.
```

```suggestion
   for the steps to install Homebrew on macOS.
```

or "for the installation instructions for Homebrew  ... "

---

Agreed. Updating.